src/FakeItEasy/Core/FakeManager.PropertySetterRule.cs

```diff
@@ -32,17 +32,34 @@ public void Apply(IInterceptedFakeObjectCall fakeObjectCall)
             {
                 Guard.AgainstNull(fakeObjectCall, nameof(fakeObjectCall));
 
-                var newRule = new CallRuleMetadata
-                                  {
-                                      CalledNumberOfTimes = 1,
-                                      Rule = new PropertyBehaviorRule(fakeObjectCall.Method, this.fakeManager)
-                                                 {
-                                                     Indices = fakeObjectCall.Arguments.Take(fakeObjectCall.Arguments.Count - 1).ToArray(),
-                                                     Value = fakeObjectCall.Arguments.Last()
-                                                 }
-                                  };
-
-                this.fakeManager.AllUserRules.AddFirst(newRule);
+                //// Setting the property adds a PropertyBehaviorRule to store the assigned value
+                //// The PropertyBehaviorRule is added at the end of user rules to avoid overriding
+                //// explicit configurations of the property made with CallTo.

```

Comment:
I guess StyleCop is complaining about the blank line after the comment when it only has `//`.

What does the comment apply to? The next line of code or the whole method? (This is exactly the ambiguity that StyleCop does't like.) If it's the next line, then the line break should be removed. If it's the whole method, then it should move out to above the method signature.